src/docfx/SubCommands/DependencyCommandCreator.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.SubCommands
+{
+    using Microsoft.DocAsCode;
+    using Microsoft.DocAsCode.Plugins;
+
+    [CommandOption("dependency", "Export dependency file")]
```

Comment:
can we define a subcommand as `export`? I think there might be other cases we want to add into the `dependency` command